src/test/tensorBoard/tensorBoardSession.test.ts

```diff
@@ -31,7 +32,13 @@ suite('TensorBoard session creation', async () => {
         sandbox.stub(ExperimentHelpers, 'inDiscoveryExperiment').resolves(false);
         // Pretend to be in experiment
         const experimentService = serviceManager.get<IExperimentService>(IExperimentService);
-        sandbox.stub(experimentService, 'inExperiment').resolves(true);
+        const inExperimentStub = sandbox.stub(experimentService, 'inExperiment');
+        inExperimentStub.resolves(true);
+
+        // These are used in interpreter service
+        inExperimentStub.withArgs(DiscoveryVariants.discoverWithFileWatching).resolves(false);
+        inExperimentStub.withArgs(DiscoveryVariants.discoveryWithoutFileWatching).resolves(false);
```

Comment:
Would stubbing `inDiscoveryExperiment` directly work?

```suggestion
        sandbox.stub(experimentService, 'inExperiment').resolves(true);
        sandbox.stub(ExperimentHelpers, 'inDiscoveryExperiment').resolves(false);
```

(still needs `import * as ExperimentHelpers from '../../client/common/experiments/helpers';`)

Reply:
If you look at the code it is already there. I will try removing the part I added.